control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -128,10 +128,10 @@ func (r Reconciler) reconcileConsumersInPlacement(
 	}
 
 	// Stable sort consumers so that we give consumers different deletion
-	// priorities based on their state.
+	// priorities based on their state (readiness, etc).
 	//
-	// Consumers at the tail of the list are deleted.
-	sort.SliceStable(consumers, func(i, j int) bool { return consumers[i].IsLessThan(consumers[j]) })
+	// Consumers at the tail of the list are deleted first.
```

Comment:
What does it mean by deleted first here? In the example in `[consumer_types_test.go](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1773/files#diff-7295594e985aadb86147d94b6b45a5f2bef68d0a79694ca41f5ffaf12ba70379), isn't `c1` deleted first before `c2` based on the for loop below?

Reply:
+1.
```suggestion
	// Consumers at the tail of the list are deleted.
```